inductiva/_cli/cmd_projects/download.py

```diff
@@ -0,0 +1,80 @@
+"""Download the project tasks files via CLI."""
+import concurrent.futures as cf
+import argparse
+
+import inductiva
+
+
+def download_projects(args):
+    """Download the project tasks files.
+
+    This function downloads the files of the tasks of a project.
+    Args:
+        args.project_name (str): Name of the project to download.
+        args.output_dir (str): Directory to save the downloaded files 
+            default(inductiva-output).
+        args.files (list): List of files to download.
+        args.std (bool): Flag to download the standard output and error files.
+    The downloads are done in parallel using a ThreadPoolExecutor with a maximum
+    of 10 workers.
+    """
+
+    project_name = args.project_name
+    output_dir = args.output_dir
+    files = args.files
+    std = args.std
+
+    if std:
+        files = ["stdout.txt", "stderr.txt"]
+
+    project = inductiva.projects.Project(project_name)
+    project_tasks = project.get_tasks()
+
+    if output_dir is not None:
+        inductiva.set_output_dir(output_dir)
+
+    futures = []
+
+    with cf.ThreadPoolExecutor(max_workers=10) as executor:
+        for task in project_tasks:
+            future = executor.submit(task.download_outputs, filenames=files)
```

Comment:
This makes a lot of text into the terminal
![Screenshot 2024-06-18 at 09 50 36](https://github.com/inductiva/inductiva/assets/67326251/565de9e0-8533-472a-8e5b-b4cd04dd556e)

Should we wrap this ThreadPool into a tqdm?

Reply:
This behaviour is defined in the `task.download_output`
We have a progress bar if we download all files.